### ISSUE TITLE
Jet Menu Custom Icons

### ISSUE THREAD

**Issue by stasonua0** [CrocoBlock/suggestions#2398]
Hello, consider choosing here https://prnt.sc/wf9f21 custom icons from the Elementor set

/*Оригинал*/
Здравствуйте, рассмотрите возможность выбора тут https://prnt.sc/wf9f21 кастомных иконок из набора Elementor

**Comment by honlapdavid** [comment]
+1

**Comment by leegaughanbrown** [comment]
Yes. It would be great if we could choose from any/all the icon sets that we have been imported in to Elementor.

**Comment by aalexandr21** [comment]
Unfortunately, we do not plan to introduce such functionality into our plugin. Now our team is working on more priority tasks. However, in the future, we will not exclude the possibility of adding this functionality to the core of our plugin.

Now you can use custom SVG icons there.

Thank you for understanding.
